docs/manual-testing-playbook.md

```diff
@@ -0,0 +1,997 @@
+# Manual testing playbook
+
+This document serves to describe the state of tailscale as packaged in the snap,
+and to facilitate repeatable manual testing.
+
+There are many shell code snippets in this document,
+designed for you to be able to copy and paste into your shell.
+Assuming the following prerequisites are met,
+you should be able to follow the document,
+running the snippets in order,
+to verify everything described here.
+
+### Prerequisites
```

Comment:
The level of "Prerequisites" seems incorrect.

![image](https://github.com/user-attachments/assets/f4425822-7db8-43a9-be5b-51023d74db36)

Reply:
Hmm yeah I wasn't sure about where to put that. I might keep it the same level, but move it under "setup"?

Reply:
Updated; please let me know what you think. :)